src/test-helpers/e2e-helpers/mobile/waitForEditable.ts

```diff
@@ -0,0 +1,14 @@
+import { Browser } from 'webdriverio'
+import getEditable from './getEditable'
+
+/** Wait for editable element that contains given value. */
+const waitForEditable = async(browser: Browser<'async'>, value: string) => {
+  await browser.waitUntil(async () => await browser.execute(value => {
+    return Array.from(document.getElementsByClassName('editable'))
+      .filter(element => element.innerHTML === value).length > 0
+  }, value)
+  , { timeout: 11000 })
```

Comment:
Can we defer to the containing test's timeout? Ideally helper functions should not be specifying their own timeouts.

Same for `waitForElement` and `waitForElementNotExist`